src/Codata/Guarded/Stream.agda

```diff
@@ -0,0 +1,142 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Infinite streams defined as coinductive records
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --without-K --guardedness #-}
+
+module Codata.Guarded.Stream where
+
+open import Data.List.Base using (List)
+open import Data.List.NonEmpty.Base as List⁺ using (List⁺)
```

Comment:
```suggestion
open import Data.List.Base using (List; []; _∷_)
open import Data.List.NonEmpty.Base as List⁺ using (List⁺; _∷_)
```
Agda does type-directed disambiguation of constructors so you can import all of these and use them unqualified
in the module.